draft/draft-ietf-bfd-optimizing-authentication.xml

```diff
@@ -619,21 +680,55 @@ reference:    RFC XXXX
 	  software or hardware, due to the use of additional computational
 	  resources these mechanisms use.
 	</t>
+
 	<t>
-	  The approach described in this document enhances the ability
-	  to authenticate a BFD session by taking away the onerous
-	  requirement that every BFD control packet be strongly authenticated. By
-	  strongly authenticating packets that affect the state of the session,
-	  the security of the BFD session is maintained. In this mode,
-	  packets that are a significant change but are not strongly
-	  authenticated, are dropped by the system. Therefore, a
-	  malicious user that tries to inject a non-authenticated
-	  packet; e.g. with a Down state to take a session down will
-	  fail. That combined with the proposal of using sequence number
-	  defined in <xref
-	  target="I-D.ietf-bfd-secure-sequence-numbers">Meticulous Keyed
-	  ISAAC for BFD Authentication</xref> further enhances the
-	  security of BFD sessions.
+	  The optimized procedures in this document provide a different level of
+	  resistance to attack than methods using a single authentication
+	  mechanism:
+	  <ul>
+	    <li>
+	      The more computationally intensive authentication mechanisms used
+	      for optimized authentication are expected to have similar
+	      cryptographic strength acceptable for BFD use as mechanisms that
+	      do not use optimized authentication.
```

Comment:
I am confused by this statement. How can computationally intensive authentication have similar strengths to not using (optimized, or for that matter any) authentication? Or did you mean "mechanisms that use BFD Authentication as defined in RFC 5880".